tests/test_routes.py

```diff
@@ -0,0 +1,63 @@
+from app.models.planet import Planet
+
+def test_get_all_planets_returns_empty_list_when_db_is_empty(client):
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == []
+    
+def test_all_planets_return_a_list_of_planets(client, two_planets):
+    response = client.get("/planets")
+    response_body = response.get_json()
+    
+    assert response.status_code == 200
+    assert response_body == [{
+        "id": 1,
+        "name": "Jupiter",
+        "description": "Jupiter has sixty-seven moons",
+        "distance_from_sun": "48.4"
+    },
+        {
+        "id": 2,
+        "name": "Mercury",
+        "description": "Mercury has zero moons",
+        "distance_from_sun": "3.5"
+    }]
+
+
+    
+def test_get_one_planet_returns_seeded_planet(client, two_planets):
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+    
+    assert response.status_code == 200
+    assert response_body["id"] == 1
+    assert response_body["name"] == "Jupiter"
+    assert response_body["description"] == "Jupiter has sixty-seven moons"
+    assert response_body["distance_from_sun"] == "48.4"
```

Comment:
Would be good to compare the response_body with what was saved in the database when the fixture ran instead of comparing the response_body to string literals. 

If you refactored two_planets to return a list of saved planet instances, then you can refactor the test. See my comment in test_all_planets_return_a_list_of_planets as an example.